jvb/src/main/kotlin/org/jitsi/videobridge/dcsctp/DcSctpTransport.kt

```diff
@@ -127,7 +132,9 @@ abstract class DcSctpBaseCallbacks(
     }
 
     override fun OnStreamsResetPerformed(outgoingStreams: ShortArray) {
-        transport.logger.info("Surprising SCTP callback: outgoing streams ${outgoingStreams.joinToString()} reset")
+        // This is normal following a call to close(), which is a hard-close (as opposed to shutdown() which is
+        // soft-close)
+        transport.logger.info("Outgoing streams ${outgoingStreams.joinToString()} reset")
```

Comment:
Is this useful at `info`?

Reply:
The sctp info logs are probably a bit chatty at this point, but that can be revisited later